plugin/chatgpt.vim

```diff
@@ -171,7 +202,7 @@ def chat_gpt(prompt):
             })
 
   if session_id:
-    content = '\n\n>>>User:\n' + prompt + '\n\n>>>Assistant:\n'.replace("'", "''")
+    content = '\n\n>>>User:\n' + prompt + '\n\n>>>Assistant:\n'.replace("'", "''")
```

Comment:
I have some concerns removing the `^A` from this, _if_ the OpenAI response includes this string, ```\n\n>>>User:\n``` it will break the parser.